playbooks/roles/cluster/tasks/bootstrap_ssh_keys.yml

```diff
@@ -0,0 +1,33 @@
+---
+- hosts: localhost
+  connection: local
+  tasks:
+    - name: Copy the tower_github_scm key to server.
```

Comment:
Can we break this copy the credential repo into its own task so we can call it from other plays?

I've already got another play where i'm going to be referencing the repo, so we could check if it exists in other plays before referencing stuff in it and then call a create the creds repo play if it doesn't.

Reply:
Yep yep

Reply:
clone_credential_repo.yml task added